lib/reek/cli/utilities.rb

```diff
@@ -0,0 +1,12 @@
+module Reek
+  module CLI
+    # CLI utilities
+    module Utilities
```

Comment:
Oh, actually the `require` part is incidental. How about `Reek::Silencer.silently { require 'x' }`?